app/assets/javascripts/float-kit/addon/components/d-menu.gjs

```diff
@@ -54,18 +42,17 @@ export default class DMenu extends Component {
     };
   }
 
-  @action
-  allowedProperties() {
```

Comment:
I don't remember why I made it a function at some point, there must have been a reason but I agree a getter is better :+1: